experiments/ClimaEarth/components/shared/restore.jl

```diff
@@ -0,0 +1,77 @@
+# Define shared methods to allow reading back a saved cache
+
+import ClimaComms
+import ClimaCore
+import ClimaCore: DataLayouts, Fields, Geometry
+import ClimaCore.Fields: Field, FieldVector, field_values
+import ClimaCore.DataLayouts: AbstractData
+import ClimaCore.Geometry: AxisTensor
+import ClimaCore.Spaces: AbstractSpace
+import ClimaUtilities.TimeVaryingInputs: AbstractTimeVaryingInput
+import StaticArrays
+import NCDatasets
+
+function restore!(v1::T1, v2::T2, comms_ctx; name = "", ignore) where {T1, T2}
```

Comment:
Can you add more documentation of the expected behavior of this function?

Reply:
Added